### PR TITLE
y*: touch-up various formulae

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/y/yank.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def install
   end
 
   test do
-    (testpath/"test.exp").write <<~EOS
+    (testpath/"test.exp").write <<~EXPECT
```
Comment (chenrui333) [inline Formula/y/yank.rb:29]: any refs for this, EXPECT does not do syntax highlighting for me.

Comment (cho-m) [reply Formula/y/yank.rb:29]: Could use TCL instead as mainly an extension on top of that.

Comment (chenrui333) [reply Formula/y/yank.rb:29]: is there any issue on ruby-lsp for these languages support?

Comment (cho-m) [reply Formula/y/yank.rb:29]: I think it is mainly a PR situation.

https://github.com/Shopify/ruby-lsp/blob/main/vscode/grammars/ruby.cson.json shows support for HTML, HAML, XML, SQL, GQL/GRAPHQL, CSS, CPP, C, JS/JAVASCRIPT, JQUERY, SH/SHELL, LUA, RUBY, YAML/YML, SLIM

**Review by branchv — APPROVED** [review]

File: Formula/y/yank.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/y/yank.rb:29]: make sense 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**